### ISSUE TITLE
{Linked:Bug11089145} [Adaptive Cards > Designer] - Narrator is not announcing the descriptive information of the 'Line Items' and 'Edit Additional Details' Expand or Collapse buttons.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8699]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/explorer/Action.ToggleVisibility.html
OS Version: 23H3(OS Build 25951.1010)
Browser Version: Edge Dev 117.0.2033.0
Screen Reader: Narrator

**Pre-Requisite:**
1. Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Navigate the screen and observe the issue.

**Actual Result:** 

Narrator is not announcing the descriptive information of the Expand or Collapse buttons.

Narrator is announcing as "Expand items button" and "Button Expanded".

**Observation:**

Narrator is announcing incorrect state for the edit additional details buttons.

**Expected Result:** 
Narrator should announce the descriptive information of the Expand or Collapse buttons.

Narrator should announce as "Expand line items button collapsed" and "Expand Edit Additional Details buttons collapsed".

**User Impact:**
Users who rely on screen readers will be impacted as the Narrator is not announcing the descriptive information of the Expand or Collapse buttons.

**WCAG Reference:** 
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![MAS1 3 1-Narrator is not announcing the descriptive information of the 'Line Items' and 'Edit Additional Details' Expand or Collapse buttons](https://github.com/microsoft/AdaptiveCards/assets/95913748/165f533d-f33c-4ca3-bddb-35bad60dcb89)

https://github.com/microsoft/AdaptiveCards/assets/95913748/78034694-b9c4-4427-beed-6000e3489ce6

[Test JSON.docx](https://github.com/microsoft/AdaptiveCards/files/12818372/Test.JSON.docx)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vagpt,

The card author can resolve this issue by updating the atlText of the images to be "Expand line items" and "Expand Edit Additional Details".

Let me know if this doesn't work!
Anna

**Comment by vagpt** [comment]
Adding A11yTTValidated comment as per the comment above.
